vaadin-archetype-spring-application/src/main/resources/archetype-resources/src/main/java/views/MainLayout.java

```diff
@@ -0,0 +1,116 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package}.views;
+
+import java.util.Optional;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.ComponentUtil;
+import com.vaadin.flow.component.applayout.AppLayout;
+import com.vaadin.flow.component.applayout.DrawerToggle;
+import com.vaadin.flow.component.avatar.Avatar;
+import com.vaadin.flow.component.html.Image;
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.tabs.Tab;
+import com.vaadin.flow.component.tabs.Tabs;
+import com.vaadin.flow.component.tabs.TabsVariant;
+import com.vaadin.flow.router.RouterLink;
+import com.vaadin.flow.server.PWA;
+import com.vaadin.flow.theme.Theme;
+import com.vaadin.flow.router.Route;
+import com.vaadin.flow.router.PageTitle;
+import ${package}.views.MainLayout;
+import ${package}.views.helloworld.HelloWorldView;
+import ${package}.views.about.AboutView;
+
+/**
+ * The main view is a top-level placeholder for other views.
+ */
+@PWA(name = "My App", shortName = "My App", enableInstallPrompt = false)
+@Theme(themeFolder = "${artifactId}")
```

Comment:
This needs a value in 14.x

But anyways, the other archetype produces an app with just a simple view, why this creates a full featured app, I think it should be just an skeleton like the other, for complicated apps we have the start.vaadin.com

Reply:
Fixed the themeFolder issue.
I wouldn't call this a full-featured app. I regard this as a skeleton with the basic building blocs of a Vaadin app. But in any case, we should aim to generate a consistent skeleton for Spring and plain Java projects. I'll change this one accordingly.